src/com/google/JsComp.gwt.xml

```diff
@@ -0,0 +1,12 @@
+<module rename-to="jscomp">
+  <inherits name="com.google.gwt.user.User" />
+  <inherits name="com.google.common.base.Base"/>
+  <inherits name="com.google.common.collect.Collect"/>
+  <source path="debugging/sourcemap" />
+  <source path="javascript/jscomp" excludes=".svn,.git,**/ant/**,**/refactoring/**,**/webservice/**,**/testing/**" />
+  <source path="javascript/rhino" excludes=".svn,.git,**/testing/**" />
+  <super-source path="javascript/jscomp/gwt/super" />
```

Comment:
I was asking because the submodules couldn't seem to find their super-sources.
I guess I can't use a non-canonical path like
`<super-source path="../jscomp/gwt/super" />`?